parser-typechecker/src/Unison/CommandLine/OutputMessages.hs

```diff
@@ -879,7 +879,8 @@ listOfDefinitions' :: Var v
 listOfDefinitions' ppe detailed results =
```

Comment:
Is `find` the only place `listOfDefinitions'` is used?  Guessing we don't generally want to cap every caller's results.  
Also what if there are more than `cap` definitions ending in `take`?  Yes we can refine it if we know the prefix, but what if we don't?  We shouldn't cap results without explaining how to ask for the next page / capful.  e.g. `hoogle` says:
> -- plus more results not shown, pass --count=20 to see more

Reply:
Good call - it wasn't. Was also used for listing definitions that would be impacted by a delete and maybe something else. Now that we are passing the show all flag explicitly to the `listOfDefinitions'` function, it caught all these places and I just did show all at each call site to preserve their current behavior.